js/stores/appStore.js

```diff
@@ -239,7 +239,7 @@ function windowDefaults () {
     windowOffset: 20,
     webPreferences: {
       sharedWorker: true,
-      partition: ''
+      partition: 'main-1'
```

Comment:
i think they are cacheable in theory, but they are small and relatively infrequent so i doubt it's much of a performance issue if not